diff --git a/src/software/convert/main_importE57.cpp b/src/software/convert/main_importE57.cpp
index 82c5a90e5..b871711e3 100644
--- a/src/software/convert/main_importE57.cpp
+++ b/src/software/convert/main_importE57.cpp
@@ -61,12 +61,13 @@ struct PointInfoVectorAdaptator
 using PointInfoKdTree =
   nanoflann::KDTreeSingleIndexAdaptor<nanoflann::L2_Simple_Adaptor<double, PointInfoVectorAdaptator>, PointInfoVectorAdaptator, 3>;
 
-template<typename _DistanceType, typename IndexType = size_t>
+template<typename _DistanceType, typename _IndexType = size_t>
 class BestPointInRadius
 {
   public:
     // Necessary since nanoflann 1.7.0: https://github.com/jlblancoc/nanoflann/commit/9c84c4c32c2bfa7077cc8873dd3b5bcbb557da90
     using DistanceType = _DistanceType;
+    using IndexType = _IndexType;
     const DistanceType m_radius;
     const std::vector<dataio::E57Reader::PointInfo>& m_points;
     const std::map<int, Eigen::Vector3d>& m_cameras;
