--- a/src/aliceVision/fuseCut/Kdtree.hpp
+++ b/src/aliceVision/fuseCut/Kdtree.hpp
@@ -48,11 +48,13 @@ typedef nanoflann::L2_Simple_Adaptor<double, PointVectorAdaptator> KdTreeDist;
 typedef nanoflann::KDTreeSingleIndexAdaptor<KdTreeDist, PointVectorAdaptator, 3> KdTree;
 
-template<typename _DistanceType, typename IndexType = size_t>
+template<typename _DistanceType, typename _IndexType = size_t>
 class SmallerPixSizeInRadius
 {
   public:
     // Necessary since nanoflann 1.7.0: https://github.com/jlblancoc/nanoflann/commit/9c84c4c32c2bfa7077cc8873dd3b5bcbb557da90
     using DistanceType = _DistanceType;
+    // Necessary since nanoflann 1.10: ResultSet must expose IndexType
+    using IndexType = _IndexType;
     const DistanceType radius;
 
     const std::vector<double>& m_pixSizePrepare;
